primitives/src/lib.rs

```diff
@@ -20,6 +20,8 @@ pub use zeitgeist_currencies_extension::ZeitgeistCurrenciesExtension;
 pub use zeitgeist_multi_reservable_currency::ZeitgeistMultiReservableCurrency;
 
 pub const BASE: u128 = 10_000_000_000;
+pub const MIN_LIQUIDITY: Balance = BASE * 100;
```

Comment:
```suggestion
pub const MIN_LIQUIDITY: Balance = 100 * BASE;
```

easier to parse putting the amount first